pack/sg.json

```diff
@@ -416,7 +416,7 @@
     "deck_limit": 3,
     "faction_code": "shaper",
     "faction_cost": 2,
-    "flavor": "\"You know there's no water in the Sea of Tranquility, right?\"\n\"That doesn't mean there's no beach.\"",
+    "flavor": "“You know there's no water in the Sea of Tranquility, right?”\n“That doesn't mean there's no beach.”",
```

Comment:
Is this intentional?

Reply:
yep.  the v2/printing version had fancy quotes.

Reply:
As far as I can tell there's no difference between the quote marks on VRcation and other flavour text in Gateway. Should v2 be changed to be consistent rather than this changed to match v2?

Reply:
No other card with quotes uses fancy quotes. I don't think we should be using them here either.

Reply:
Maybe I misunderstood your comment that you've reverted this change, in which case my apologies.